app/src/main/java/com/hsLink/hslink/data/dto/request/PostRequestDto.kt

```
@@ -0,0 +1,4 @@
+package com.hsLink.hslink.data.dto.request
+
+class PostRequestDto {
+}
```

detekt (1.23.8)

[warning] The class or object PostRequestDto is empty.

(detekt.empty-blocks.EmptyClassBlock)